### PR TITLE
UnmapSharedMemory

### PR DESCRIPTION
**Pull request by N00byKing**
This is my fist time doing actual work on the Emulator without guidelines such as the pretty much now finished Meta-Issue 40. As such, I advise reviewers to be especially critical.

### CODE REVIEW

**Comment by N00byKing** [comment]
yup

**Comment by bunnei** [comment]
This isn't a stub, right?

**Comment by N00byKing** [comment]
@bunnei now im confused. Could you clear up the terminology for me?

**Comment by mailwl** [comment]
may i answer?
stub mean "do nothing". real work not stub, just remove this word to not confuse developers

**Comment by N00byKing** [comment]
Oh, i thought of stub as in "unfinished" or "partly done".

File: src/core/hle/kernel/svc.cpp [hunk 1]
```diff
@@ -443,6 +443,15 @@ static ResultCode MapSharedMemory(Handle shared_memory_handle, VAddr addr, u64 s
     return RESULT_SUCCESS;
 }
 
+static ResultCode UnmapSharedMemory(Handle shared_memory_handle, VAddr addr, u64 size) {
+    LOG_WARNING(Kernel_SVC, "called, shared_memory_handle=0x%08X, addr=0x%llx, size=0x%llx",
```
Comment (Subv) [inline src/core/hle/kernel/svc.cpp]: Use PRIX64 or PRIx64 instead of llx

Comment (N00byKing) [reply src/core/hle/kernel/svc.cpp]: done

Comment (N00byKing) [reply src/core/hle/kernel/svc.cpp]: nvm .-.

Comment (N00byKing) [reply src/core/hle/kernel/svc.cpp]: k now it works

**Pull request merged**